src/rowtable.typ

```diff
@@ -152,7 +153,7 @@
 #let _as-equation(elt, block: false) = {
   // if we get a table cell here, we need to invert the order
   // so that the cell is the outermost layer
-  if isfunc(elt, table.cell) {
```

Comment:
wonder if you should pass down the function you're using instead

Reply:
it doesn't matter much either way I think. We kind of help the user in the current state of the PR (they will find their incorrect use of grid vs table cell faster if we use `iscell` here).

I like it better as a simple transformation of the input, unfortunately, the code has passed the line for "simple".

Reply:
okay.